ea_findonemidpoint.m

```diff
@@ -1,7 +1,24 @@
 function [numidpoint,greyobj,options]=ea_findonemidpoint(slicebw,estpoint,mask,options)
-% __________________________________________________________________________________
-% Copyright (C) 2014 Charite University Medicine Berlin, Movement Disorders Unit
-% Andreas Horn
+%
```

Comment:
Please provide a description of the functions, as well as the inputs and outputs